src/views/CreateOrUpdateRequest.svelte

```diff
@@ -180,7 +202,7 @@ async function cancelRequest() {
     
     {#if !isNew}
       <div class="col-auto text-center">
-        <button on:click={cancelRequest} class="btn btn-outline-danger">
+        <button on:click|preventDefault={cancelRequest} class="btn btn-outline-danger">
```

Comment:
you won't need to prevent the "submit" here if you either make this an "button" type input or a link that looks like a button.